gemini-resin-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/resin.xml

```diff
@@ -35,7 +35,7 @@
       <filter-mapping url-pattern='*' filter-name="gzip" />
     </web-app-default>
 
-    <resin:import path="${RESIN_HOME}/conf/app-default.xml" />
+    <resin:import path="${resin.home}/conf/app-default.xml" />
```

Comment:
This may no longer be true, but I believe that I had `RESIN_HOME` here because 1. Eclipse did not support passing through environment variables to the java execution and 2. did not support dot-notated vars in the configuration dialog. Ignore if this just works in Eclipse now.